code/src/MessageList.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react'
+import { LikeButton } from "./LikeButton"
+import moment from "moment"
+import './message-list.css'
+
+export const MessageList = () => {
+  const messages_url = "https://technigo-thoughts.herokuapp.com/"
+  const [messages, setMessages] = useState([])
+
+  //Gets messages from API
+  useEffect(() => {
+    fetch(messages_url)
+      .then(res => res.json())
+      .then(data => {
+        setMessages(data)
+      })
+  }, [])
+
+  //Shows likes on thoughts
+  const onMessageLiked = (likedMessageId) => {
+    const updatedMessages = messages.map((message) => {
+      if (message._id === likedMessageId) {
+        message.hearts += 1
+      }
+      return message
+    })
+    setMessages(updatedMessages)
+  }
+
+  //Returns an article with a thought, a like-button and a time-stamp
+  return (
+    <section className="message-list">
+      {
+        messages.map(message => (
+          <article className="message" key={message._id}>
```

Comment:
How come this is on one line, but in the other places with more than one prop they are on separate lines?